lib/commands.js

```diff
@@ -3113,4 +3113,19 @@ commands.touchId = function(){
   });
 };
 
+/**
+ * touchId(cb) -> cb(err, touchId)
+ * 
+ * @jsonWire GET /session/:sessionId/appium/simulator/touch_id
```

Comment:
Is this a `GET` or a `POST` request? It says the former here but the actual call is the latter, which seems more reasonable.